src/components/DiffView/index.tsx

```diff
@@ -97,13 +97,18 @@ export const trimHunkChanges = (
   return trimmed;
 };
 
+export const changeCanBeCommentedUpon = (change: ChangeInfo) => {
+  return ['insert', 'normal'].includes(change.type);
```

Comment:
It would be a bit safer to use [`ChangeType`](https://github.com/mozilla/addons-code-manager/blob/55888e7084550467fcdfa0de14aca504bdd5d318/src/%40types/react-diff-view/index.d.ts#L4):

```tsx
['insert' as ChangeType, 'normal' as ChangeType].includes(change).type
```

Reply:
Sorry, what I meant was to try and make it more type safe (I don't think the code above will do that). Maybe something like this?

```tsx
const types: ChangeType[] = ['insert', 'normal'];
return types.includes(change.type);
```

Reply:
This is a good idea, thanks. I wonder, when looking at this, whether it would be better for `ChangeType` to be an `enum`? Perhaps that would be a problem for `react-diff-view`?

Reply:
Yeah, making `ChangeType` an enum would probably help (maybe save that for another patch as it could produce a big diff).

Reply:
Thanks. I filed #1322 as a follow-up.